### DISCUSSION TITLE
Anchor Dropdown/Popover content to center of screen

### DISCUSSION THREAD

**Discussion by vnugent** [radix-ui/primitives#1761] [Help] [answered]
On mobile I want to make Dropdown or Popover content to always show up in the middle of the screen.  Is it possible?

Example: IG 
---
<img width="307" alt="Screen Shot 2022-11-06 at 6 39 14 PM" src="https://user-images.githubusercontent.com/3805254/200186243-93d278b8-3653-4ec2-a9f2-c109346a265f.png">

**Comment by benoitgrelard** [comment, marked as answer]
Isn't this a `Dialog`?
